### ISSUE TITLE
Error while building in map_builder_std.cpp:568:25: error: 'game_time' has not been declared

### ISSUE THREAD

**Issue by FredBezies** [martin-tornqvist/ia#202]
I was trying to get it build, but I got this error: 

```
/home/fred/infra-arcana-git/src/ia/src/map_builder_std.cpp: In member function 'virtual bool MapBuilderStd::build_specific()':
/home/fred/infra-arcana-git/src/ia/src/map_builder_std.cpp:568:25: error: 'game_time' has not been declared
                         game_time::add_mob(event);
                         ^~~~~~~~~
make[3]: *** [CMakeFiles/ia.dir/build.make:1455: CMakeFiles/ia.dir/src/map_builder_std.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/Makefile2:68: CMakeFiles/ia.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:80: CMakeFiles/ia.dir/rule] Error 2
make: *** [Makefile:164: ia] Error 2
==> ERROR: A failure occurred in build().
```

**Comment by martin-tornqvist** [comment]
Ah, I see what's going on. There are some headers which are only included for debug builds (they are inside an "#ifndef NDEBUG" condition).

I got the same error now when I tried building the release version of the game. I'll fix it.

**Comment by martin-tornqvist** [comment]
Fixed: 3763d65b3f587c7bf66a13460bbbdd9f9a01ec77

**Comment by FredBezies** [comment]
Thanks!
